composer.json

```diff
@@ -32,7 +32,9 @@
         "drupal/externalauth": "1.2.0",
         "drupal/field_group": "3.0",
         "drupal/layout_builder_restrictions": "2.5",
-        "drupal/layout_builder_styles": "1.0-beta1"
+        "drupal/layout_builder_styles": "1.0-beta1",
+        "drupal/migrate_plus": "5.1",
```

Comment:
can these be require-dev?

Reply:
`migrate_plus` is a required dependency as it contains the plugins to import from a serialized URL/file and can't be optional.

`migrate_tools` could be a dev dependency, but composer only installs dev dependencies for the root package (which in our case is the scaffolding, not this installation profile). This is one reason we don't currently have dev dependencies in this composer.json. That's something we're looking into as part of #33 .

Reply:
Oh,makes sense